Khan/graphie-to-png
# Switch the math renderer from KaTeX to MathJax

## Summary:
This is accomplished by adapting the MathJaxRenderer interface to look
like KaTeX, since that's what Khan Exercises expects.

Issue: https://khanacademy.atlassian.net/browse/LC-1002

Test plan:

- Start the dev server: `./server.sh`.
- Visit http://localhost:8765
- View all the example graphies by selecting them from the dropdown.
  They should all look pretty much the same as the renderings on
  production (http://graphie-to-png.khanacademy.systems/)
- Click the "Convert to Image" button. Copy the URL that appears.
- Paste the URL back into a new graphie editor window. You should get the same rendered image and labels.
- Paste the URL into a Markdown image link in the [exercise editor](https://www.khanacademy.org/devadmin/content/exercises/interpreting-expressions/x348258ef/x53040f6b0fa6ee1e), e.g. `![](web+graphie://...)`. The image and labels should render correctly in the preview.

<img width="1168" alt="Screen Shot 2023-11-30 at 4 28 02 PM" src="https://github.com/Khan/graphie-to-png/assets/693920/412e4740-e7f2-43dd-9a50-fadd41948603">
